csi/controller.go

```diff
@@ -48,6 +50,13 @@ func (s *OsdCsiServer) ControllerGetCapabilities(
 			},
 		},
 	}
+	capListVolumes := &csi.ControllerServiceCapability{
+		Type: &csi.ControllerServiceCapability_Rpc{
+			Rpc: &csi.ControllerServiceCapability_RPC{
+				Type: csi.ControllerServiceCapability_RPC_LIST_VOLUMES,
+			},
+		},
+	}
 
 	//
 	// Here we will be adding the following in future patches:
```

Comment:
could remove the comment about adding List Volumes Capability as we are adding it now